gtsam/hybrid/tests/testHybridNonlinearFactorGraph.cpp

```diff
@@ -857,21 +857,23 @@ namespace test_relinearization {
  */
 static HybridNonlinearFactorGraph CreateFactorGraph(
     const std::vector<double> &means, const std::vector<double> &sigmas,
-    DiscreteKey &m1, double x0_measurement) {
+    DiscreteKey &m1, double x0_measurement, double measurement_noise = 1e-3) {
   auto model0 = noiseModel::Isotropic::Sigma(1, sigmas[0]);
   auto model1 = noiseModel::Isotropic::Sigma(1, sigmas[1]);
-  auto prior_noise = noiseModel::Isotropic::Sigma(1, 1e-3);
+  auto prior_noise = noiseModel::Isotropic::Sigma(1, measurement_noise);
 
   auto f0 =
       std::make_shared<BetweenFactor<double>>(X(0), X(1), means[0], model0);
   auto f1 =
       std::make_shared<BetweenFactor<double>>(X(0), X(1), means[1], model1);
 
   // Create HybridNonlinearFactor
+  // We multiply by -2 since the we want the underlying scalar to be log(|2πΣ|)
```

Comment:
I think that's wrong.

Reply:
According to the math, this is correct, though it is another way of doing things.
We can hide $\sqrt{\log(|2\pi \Sigma|)}$ so that on squaring and halving, we get $\frac{1}{2}\log(|2\pi \Sigma|) = \log(\sqrt{|2\pi \Sigma|})$.

However, since we've decided to premultiply by 2 right before hiding (inside the `augment` method), then yeah I'll have to pass in $\log(\sqrt{|2\pi \Sigma|})$.